src/components/Panel/Panel.tsx

```diff
@@ -34,9 +36,12 @@ const Panel = (props: Props) => {
   )
 
   return (
-    <Card border={color}>
+    <Card className={className ? className.concat('', '') : className} border={color}>
```

Comment:
We should be able to just pass `className` to this prop so it could simply be `className={className}`. If `className` prop is undefined it wont' have any impact.